debug_prompt.cc

```diff
@@ -0,0 +1,135 @@
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "gemma.h"
+#include "nlohmann/json.hpp"
+// copybara:import_next_line:gemma_cpp
```

Comment:
This is no longer necessary for gemma headers, only for sentencepiece. Please remove both here :)

Reply:
Done